src/LanguageServer/IdePurescript/Config.purs

```diff
@@ -198,12 +198,10 @@ formatter :: ConfigFn Formatter
 formatter =
   getString "formatter" ""
     >>> case _ of
-        "none" -> NoFormatter
-        "" -> NoFormatter
         "purty" -> Purty
         "purs-tidy" -> PursTidy
```

Comment:
Do we really need the `purs-` prefix?

Reply:
You tell me. Is `tidy` the more common name?

The npm package, the cli command and the announcement all use `purs-tidy`.

My preferred way would be to allow both `tidy` and `purs-tidy` since it's easy to mixup.

Reply:
As far as I'm aware that's its name.

Reply:
Alright then

Reply:
(interestingly both wclr and nwolverson you referred to it as tidy in your above comments :D)